packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -141,7 +141,7 @@ export const NewsFeedInput = React.forwardRef<
     const selectedNetwork = useSelectedNetworkInfo();
     const selectedNetworkId = selectedNetwork?.id || "teritori";
     const selectedWallet = useSelectedWallet();
-    const userId = getUserId(selectedNetworkId, selectedWallet?.address);
+    const userId = daoId ? daoId : selectedWallet?.userId;
```

Comment:
```suggestion
    const posterId = daoId ? daoId : selectedWallet?.userId;
```
let's make this ultra explicit

Reply:
[ec11c14](https://github.com/TERITORI/teritori-dapp/pull/1336/commits/ec11c14bd8775973edf5da3029e04310db2f9783)